### PR TITLE
[IMP] theme_*: review s_cta_box occurrences

### PR DESCRIPTION
**Pull request by chgo-odoo**
*: anelusia, artists, avantgarde, aviato, beauty, bewise, bistro, bookstore, buzzy, clean, cobalt, enark, graphene, kea, kiddo, loftspace, monglia, nano, notes, odoo_experts, orchid, paptic, real_estate, treehouse, vehicle, yes, zap

- requires https://github.com/odoo/odoo/pull/175703

task-4094396
Part of task-4077427

### CODE REVIEW

File: theme_treehouse/views/snippets/s_cta_box.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+<odoo>
+
+<template id="s_cta_box" inherit_id="website.s_cta_box">
+    <!-- Card -->
+    <xpath expr="//div[hasclass('card')]" position="attributes">
+        <attribute name="class" add="o_cc3" remove="o_cc4" separator=" "/>
+    </xpath>
+    <!-- Title -->
+    <xpath expr="//h2" position="replace" mode="inner">
+       Help us<br/>shaping the future
```
Comment (sobo-odoo) [inline theme_treehouse/views/snippets/s_cta_box.xml:11]: "shaping" => shape (no -ing with help) :+1:

File: theme_treehouse/views/snippets/s_cta_box.xml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    </xpath>
+    <!-- Paragraphs -->
+    <xpath expr="//p" position="replace" mode="inner">
+        Join our NGO and help us making the world a better place<br/><br/>
```
Comment (sobo-odoo) [inline theme_treehouse/views/snippets/s_cta_box.xml:15]: "making" => make or to make

**Review by sobo-odoo — CHANGES_REQUESTED** [review]
Here is my design-themes review :wink: 

EDIT: don't forget to undraft your PRs afterwards :+1:

File: theme_odoo_experts/views/snippets/s_cta_box.xml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+<template id="s_cta_box" inherit_id="website.s_cta_box">
+    <!-- Card -->
+    <xpath expr="//div[hasclass('card')]" position="attributes">
+        <attribute name="class" add="o_cc5" remove="o_cc4" separator=" "/>
+    </xpath>
+    <!-- Title -->
+    <xpath expr="//h2" position="replace" mode="inner">
+       Leverage your workflow<br/>with our services<br/>
+    </xpath>
```
[the remark this answers is not in the document]
Comment (chgo-odoo) [reply theme_odoo_experts/views/snippets/s_cta_box.xml:12]: Indeed, this is not intentional, good catch !

File: theme_kea/views/snippets/s_cta_box.xml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+<odoo>
+
+<template id="s_cta_box" inherit_id="website.s_cta_box">
+    <!-- Card -->
+    <xpath expr="//div[hasclass('card')]" position="attributes">
+        <attribute name="class" add="o_cc5" remove="o_cc4" separator=" "/>
+    </xpath>
+    <!-- Title -->
+    <xpath expr="//h2" position="replace" mode="inner">
+        Ready to explore?<br/>book your timeslot now
```
[the remark this answers is not in the document]
Comment (chgo-odoo) [reply theme_kea/views/snippets/s_cta_box.xml:11]: Let's make the editor happy, then 😄

File: theme_kea/views/snippets/s_cta_box.xml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    <xpath expr="//div[hasclass('card')]" position="attributes">
+        <attribute name="class" add="o_cc5" remove="o_cc4" separator=" "/>
+    </xpath>
+    <!-- Title -->
+    <xpath expr="//h2" position="replace" mode="inner">
+        Ready to explore?<br/>Book your time slot now
+    </xpath>
+    <!-- Paragraphs -->
+    <xpath expr="//p" position="replace" mode="inner">
+        Each timeslot includes a 1h discovery period.<br/><br/>
```
Comment (sobo-odoo) [inline theme_kea/views/snippets/s_cta_box.xml:15]: This "timeslot" should be changed too then :+1:

File: theme_bewise/views/customizations.xml [hunk 6]
```diff
@@ -421,4 +421,24 @@
     </xpath>
 </template>
 
+<!-- ======== CTA Box ======== -->
+<template id="s_cta_box" inherit_id="website.s_cta_box">
+    <!-- Title -->
+    <xpath expr="//h2" position="replace" mode="inner">
+        Your next university<br/>is just clicks away
```
Comment (sobo-odoo) [inline theme_bewise/views/customizations.xml:428]: I think the way I wrote it was confusing, but I just meant to add a "s" at the end of "click" :slightly_smiling_face: It was better with "a few" before :+1: 
(Sorry, I am used to quote like that for commit messages, so the dev knows where the text is, but it was useless here)

**Review by sobo-odoo — CHANGES_REQUESTED** [review]
Last details :+1:

File: theme_bewise/views/customizations.xml (hunk 6 above)
Comment (chgo-odoo) [reply theme_bewise/views/customizations.xml:428]: Ah yes sorry, my bad 😅

**Review by sobo-odoo — APPROVED** [review]
@robodoo r+

**Review by sobo-odoo — APPROVED** [review]
@robodoo r+

**Review by sobo-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**